distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -1679,6 +1683,102 @@ public void testAccessGroups() throws Exception {
     }
   }
 
+  @Test
+  public void testSecurityManagerPolicy() {
+    // TODO: tbatie - 10/12/1 - Because the security policy currently allows access to anything
```

Comment:
❓ Could we keep them on but put an Assume that would check if the current directory is under user.home in order to skip them? That way, they would run for those (and Jenkins) that do not have their workspace under user.home.

Reply:
I'd prefer to wait until the policy is more restricted to avoid unstable builds.